plugins/inputs/aliyuncms/discovery.go

```diff
@@ -63,15 +63,13 @@ func getRPCReqFromDiscoveryRequest(req discoveryRequest) (*requests.RpcRequest,
 	}
 
 	ptrV := reflect.Indirect(reflect.ValueOf(req))
-
```

Comment:
Could you please move formatting cleanup to an own PR to ease reviews! It's hard if functional changes and pure style changes are mixed and some files are only style changes...